packages/lesswrong/server/callbacks/userCallbacks.tsx

```diff
@@ -79,6 +83,46 @@ getCollectionHooks("Users").editSync.add(function maybeSendVerificationEmail (mo
   }
 });
 
+getCollectionHooks("Users").updateBefore.add(async function updateProfileTagsSubscribesUser(data, {oldDocument, newDocument}: UpdateCallbackProperties<DbUser>) {
+  // check if the user added any tags to their profile
+  const tagIdsAdded = newDocument.profileTagIds?.filter(tagId => !includes(oldDocument.profileTagIds || [], tagId)) || []
```

Comment:
Does this need to use `includes` from `underscore`?  I think this can just use the native `Array.includes()`:
```
const tagIdsAdded = newDocument.profileTagIds?.filter(tagId => !!oldDocument.profileTagIds?.includes(tagId)) ?? []
```

Reply:
I'd prefer to use `Array.includes()` as well, so I'll go ahead and change these